pwiz_tools/Skyline/TestTutorial/AbsoluteQuantTutorialTest.cs

```diff
@@ -49,16 +50,25 @@ public void TestAbsoluteQuantificationTutorial()
             // Set true to look at tutorial screenshots.
             // IsPauseForScreenShots = true;
 
-            ForceMzml = true;   // Mzml is ~8x faster for this test.
-
+            ForceMzml = false;   // Mzml is ~8x faster for this test.

```

Comment:
We don't actually want to do this for the main test implementation. It is quite a bit slower to use the .raw files. If we need this for the tutorial screenshots, then this should be based on whether PauseForScreenShots is on.